04_Chatbot/00_Chatbot_Claude.ipynb

```diff
@@ -511,9 +471,6 @@
     "from langchain.indexes.vectorstore import VectorStoreIndexWrapper\n",
     "from langchain.vectorstores import FAISS\n",
     "\n",
-    "s3_path = f\"s3://jumpstart-cache-prod-us-east-2/training-datasets/Amazon_SageMaker_FAQs/Amazon_SageMaker_FAQs.csv\"\n",
```

Comment:
We cannot keep the downloaded file as part of the repo hence downloading, can you please revert this change and download the file from source.